data-serving/data-service/api/dataserver.api.yaml

```diff
@@ -0,0 +1,54 @@
+openapi: 3.0.3
```

Comment:
There's this neat extension, in case you hadn't seen:

https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi

Reply:
nice! i was using their web editor (https://editor.swagger.io/) but in the IDE is even better!